contributors/devel/sig-testing/testing-strategy.md

```diff
@@ -0,0 +1,99 @@
+## Defining a Robust Testing Strategy
+
+This document outlines a testing strategy for Kubernetes features based on the **testing pyramid**, with considerations on the existing CI system's characteristics.
+
+### The Testing Pyramid
+
+Prioritize tests based on the testing pyramid, refer to the [Testing Guide](./testing.md):
+
+- **Unit Tests:**  The foundation. Fast, isolated, and cover individual components.
+- **Integration Tests:** Verify interactions between components within your subsystem.
+- **E2E Tests:**  Test the entire system, including interactions with external dependencies. These are the most expensive and prone to flakiness.
+
+### CI Job Types
+
+The Kubernetes job uses [prow](https://prow.k8s.io) to implement the CI system. We can differentiate between different types of jobs:
+
+- **Presubmit:** Runs before code is merged.
+    - **Blocking:** Prevents merging if tests fail. Use cautiously due to potential project-wide impact. We aim to have a very high bar on these jobs and ask for proof
+    of stability, reliability and performance.
+    - **Non-Blocking/Informational:** Provides feedback without blocking merges.
+- **Periodic:** Runs at scheduled intervals. Ideal for monitoring trends and catching regressions.
+- **Postsubmit:** Runs after code is merged. Useful for building artifacts.
+
+#### SIG-Release Blocking and Informing jobs
+
+SIG-release maintains two sets of jobs that decide whether the release is
```

Comment:
```suggestion
SIG-Release maintains two sets of jobs that decide whether the release is
```

Reply:
Or even better, [without](https://github.com/kubernetes/sig-release?tab=readme-ov-file#sig-release) the hyphen?

Reply:
Sure. In which case we should really capitalize release.
If we drop the hyphen, we should do so from the heading as well (SGTM).